### PR TITLE
Issue/1763

### PR DESCRIPTION
**Pull request by lc-thomasberger**
Move pageLabel outside of wrapper element.
Add globals for trickle label.

### CODE REVIEW

File: src/core/js/views/pageView.js [hunk 1]
```diff
@@ -17,6 +17,17 @@ define([
...
+            var accessibility = Adapt.config.get('_accessibility');
+            if (accessibility._isEnabled || accessibility._isEnabledOnTouchDevices) {
+                // create aria-label outside of #wrapper
+                this.pageLabel = document.createElement('div');
+                this.pageLabel.setAttribute('class', 'aria-label relative a11y-ignore-focus prevent-default')  ;
+                this.pageLabel.setAttribute('tabindex', 0);
+                this.pageLabel.setAttribute('role', 'region');
+                this.pageLabel.textContent = Adapt.course.get('_globals')._accessibility._ariaLabels.pageEnd;
+                document.body.appendChild(this.pageLabel);
+            }
```
Comment (oliverfoster) [inline src/core/js/views/pageView.js]:
```javascript
$("<div>", {
  class: 'aria-label relative a11y-ignore-focus prevent-default',
  tabindex: 0,
  role: 'region'
})
.html(Adapt.course.get('_globals')._accessibility._ariaLabels.pageEnd)
.appendTo($("body"));
 ```
?

Comment (lc-thomasberger) [reply src/core/js/views/pageView.js]: Yes, I know, native should be save though?
If you prefere it due to legibility I am fine to update it.

Comment (oliverfoster) [reply src/core/js/views/pageView.js]: Can you update it? It's a 49% character saving (including whitespace)? ;)

Comment (lc-thomasberger) [reply src/core/js/views/pageView.js]: well that's an argument. Sure will update it.

File: src/core/js/views/pageView.js [hunk 2]
```diff
@@ -55,7 +55,7 @@ define([
 
         remove: function() {
             if (this.pageLabel) {
```
Comment (oliverfoster) [inline src/core/js/views/pageView.js]: one more dollar $$$$

File: src/course/en/course.json [hunk 3]
```diff
@@ -78,6 +78,9 @@
             },
             "_resources": {
                 "resourcesEnd": "You have reached the end of the list of resources."
+            },
+            "_trickle": {
+                "incompleteContent": "There is incomplete content above. You must complete this before you can proceed in the course."
```
Comment (tomgreenfield) [inline src/course/en/course.json]: Could this be updated to `proceed through the course` please?

File: src/core/js/views/pageView.js [hunk 4]
```diff
@@ -17,6 +17,17 @@ define([
             this.disableAnimation = Adapt.config.has('_disableAnimation') ? Adapt.config.get('_disableAnimation') : false;
             this.$el.css('opacity', 0);
             this.listenTo(this.model, 'change:_isReady', this.isReady);
+
+            var accessibility = Adapt.config.get('_accessibility');
+            if (accessibility._isEnabled || accessibility._isEnabledOnTouchDevices) {
```
Comment (tomgreenfield) [inline src/core/js/views/pageView.js]: Reverse logic to return early?

**Pull request merged**